app/scripts/detect-multiple-instances.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Sets up two-way communication between the
+ * mainline version of extension and Flask build
+ * in order to detect & warn if there are two different
+ * versions running simultaneously.
+ */
+
+import extension from 'extensionizer';
+
+// TODO: add IDs from other browsers
+const PROD_BUILD_ID = 'nkbihfbeogaeaoehlefnkodbefgpgknn';
+const FLASK_BUILD_ID = 'ljfoeinjpaedjfecbmggjgodbgkmjkjk';
+
+const MESSAGE_TEXT = 'isRunning';
+
+/**
+ * Handles the ping message sent from other extension.
+ */
+export const onMessageReceived = (message, sender, sendResponse) => {
+  if (message === MESSAGE_TEXT) {
+    sendResponse({
+      type: 'success',
```

Comment:
Is there a specific reason for this response format? It doesn't look like the data is used anywhere.

Reply:
Let me check if I can simplify that 👀